crates/executor/src/simulate.rs

```diff
@@ -108,6 +120,9 @@ pub fn simulate(
                     fee_estimation: FeeEstimate {
                         gas_consumed: U256::from(tx_info.actual_fee.0) / gas_price.max(1.into()),
                         gas_price,
+                        data_gas_consumed: U256::from(tx_info.da_gas.l1_data_gas)
+                            / data_gas_price.max(1.into()),
```

Comment:
Probably the same issue? (`tx_info.da_gas.l1_data_gas` seems to be the gas consumption and not the fee calculated using data gas price.)